### ISSUE TITLE
Remove Historical Invalid Mapping Entries

### ISSUE THREAD

**Issue by imcoddy** [tekezo/Karabiner#449]
This might not be a bug but I just found it a bit annoying and didn't  see others bring this up so I make this as a new issue.

## Problem Desc

By using export/import function, it is a lot easier to migrate to another machine and adapt quickly, but when exporting configs to a file, the existing invalid configs will also be exported.

## Example

At first I make a new identifier in my priviate.xml as follow
```
            <item>
                <name>Swap Command_L and Option_L</name>
                <identifier>private.swap_leftcommand_and_leftoption_with_minila</identifier>
                <autogen>__KeyToKey__ KeyCode::COMMAND_L, KeyCode::OPTION_L</autogen>
                <autogen>__KeyToKey__ KeyCode::OPTION_L, KeyCode::COMMAND_L</autogen>
            </item>
```

But later I just want to organise them for the better so I change the identifier to another group.

```
            <item>
                <name>[FILCO_MINILA]Swap Command_L and Option_L</name>
                <identifier>private.filco.swap_command_l_and_option_l</identifier>
                <device_only>DeviceVendor::FILCO_MINILA,DeviceProduct::MINILA_KEYBOARD</device_only>
                <autogen>__KeyToKey__ KeyCode::COMMAND_L, KeyCode::OPTION_L</autogen>
                <autogen>__KeyToKey__ KeyCode::OPTION_L, KeyCode::COMMAND_L</autogen>
            </item>
```

Basically I just changed the name and loaded again. But when I exported again it just turned out that old identifier ```private.swap_leftcommand_and_leftoption_with_minila``` will still be exported, even it is already invalid and not found in the settings.

It is not a serious bug and everything still works fine, but if there is a way to clean up the unneeded configs it will be better, as I use git to keep the config and don't really want to see the outdated code :)

Please let me know if there's a solution for this already, thanks in advance.

**Comment by tekezo** [comment]
I've added `be_careful_to_use__clear_all_values_by_name` into command line interface at v10.6.90.
Upgrade Karabiner to the latest beta version:
https://pqrs.org/osx/karabiner/document.html#check-for-updates


Then, please execute it before import.
For example, if your profile name is `Default2`, execute the following command.
```
$ /Applications/Karabiner.app/Contents/Library/bin/karabiner be_careful_to_use__clear_all_values_by_name Default2
```

**Comment by imcoddy** [comment]
Thanks for the reply.
I just confirmed that I am running on latest stable version 10.6.0, looks like the v10.6.90 is still in the beta,   probably I will check that later on my secondary computer.
BTW, any schedule when it will become the new stable one?

**Comment by tekezo** [comment]
Karabiner-v10.6.88 and later versions is release candidate.
I would like to release soon.
